courses/management/commands/refund_enrollment.py

```diff
@@ -45,12 +51,17 @@ def handle(self, *args, **options):
                 )
             ]
 
-        self.stdout.write(
-            self.style.SUCCESS(
-                "Refunded enrollments for user: {} ({})\nEnrollments affected: {}".format(
-                    enrollment.user.username,
-                    enrollment.user.email,
-                    list(filter(bool, [program_enrollment] + run_enrollments)),
-                )
-            )
+        success_msg = "Refunded enrollments for user: {} ({})\nEnrollments affected: {}".format(
+            enrollment.user.username,
+            enrollment.user.email,
+            enrollment_summaries(filter(bool, [program_enrollment] + run_enrollments)),
         )
+
+        if enrollment.order:
```

Comment:
Can we also output a message if an order was not found for the enrollment? It would be useful to know when an order was refunded or if the enrollment for the user is not associated with an order for some reason.